postgresql-16-upgrade/scripts/migrate.sh

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env bash
+
+#####
+#
+# -- Migrate GitGuardian Database --
```

Comment:
do we need this script ? It seems it's included in `restore.sh` already

Reply:
yes, you're right, at first, I wanted to run migrations in a separate step, so I created this script. we can definitely remove it 👍